TijanaSonjic/assignment2/Q2.h

```diff
@@ -0,0 +1,88 @@
+#pragma once
+#include <iostream>
+using namespace std;
+
+class Node {
```

Comment:
We encourage you to avoid duplication - so why submit the same code twice?
This `Node` class is the same in both files.
You can submit both questions in one file if that's more convenient for you (and use one `Node` class).
Another option is to extract Node into its own file and `#include` it in both solution files.

It will also make iterations easier for you (if you want to make a change in the node class, you do not need to change it twice).

Reply:
I don't know :) I had a dilemma should I put those solution in separate files or can put all data structures in one file and just include it in both solutions. I was supposed to ask you this before submitting mu solutions.